content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -804,3 +804,63 @@ const model = openai.image('dall-e-3');
 | ---------- | ------------------------------- |
 | `dall-e-3` | 1024x1024, 1792x1024, 1024x1792 |
 | `dall-e-2` | 256x256, 512x512, 1024x1024     |
+
+## Transcription Models
+
+You can create models that call the [OpenAI transcription API](https://platform.openai.com/docs/api-reference/audio/transcribe)
+using the `.transcription()` factory method.
+
+The first argument is the model id e.g. `whisper-1`.
+
+```ts
+const model = openai.transcription('whisper-1');
+```
+
+OpenAI transcription models support a single `audio` argument which can be a `File`, `Blob`, `Uint8Array`, `ArrayBuffer`, or a `string` (a URL or base64 encoded audio data).
+
+```ts
+const model = openai.transcription('whisper-1', {
+  audio: new File([], 'audio.mp3', { type: 'audio/mp3' }),
+});
+```
+
+You can also pass additional provider-specific options using the `providerOptions` argument.
+
+```ts
+const model = openai.transcription('whisper-1', {
+  providerOptions: { openai: { timestamp_granularities: ['word'] } },
```

Comment:
we usually use `camelCase` in the provider options to match conventions (even if the provider uses `snake_case` in their apis)